### DISCUSSION TITLE
Theia Unregister Themes

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#10001] [General] [answered]
Hi,
There are few themes that aren't currently supported by our product. So, is there any option unregister such themes ? 

Thanks.

**Comment by msujew** [comment, marked as answer]
Hi @maheshrajrp, do mean the default themes (dark, light, and high constrast) or others? Themes contributed by vscode extensions can simply be disabled by removing them from the `plugins` list in your `package.json` file.

Removing the build-in themes is a bit more difficult.

**Reply by maheshrajrp** [reply]
Hi @msujew ,
Thanks for the quick reply. Built-in Themes is fine. But, I intend to remove the other themes that are listed in below screenshot. Is there any way to find the plugin that contributes these themes ? Also, does removing these plugin will affect other functionalities ? 

Thanks.


![image](https://user-images.githubusercontent.com/80001447/131476388-8dc118d2-d8f4-446f-8e75-9e330d57152f.png)

**Reply by msujew** [reply]
In your `package.json` (if you installed them with the theia command line tool) there should an entry for `vscode-builtin-theme-abyss` and `vscode-builtin-theme-defaults` which contribute these themes. These extensions don't do anything else, so you can remove them safely.
